Jenkinsfile

```diff
@@ -17,7 +17,7 @@ pipeline {
             python -m pip install --user virtualenv
             python -m virtualenv --no-site-packages .testenv
             source .testenv/bin/activate
-            pip install -r tests/requirements.txt
+            pip install -r requirements.txt
```

Comment:
These requirements are only for tests, they should remain under tests folder.

Reply:
These are the supported requirements for pas orchestrator...

Reply:
boto3 and awscli doesn't seems to be